engine/src/witness/sol/program_swaps_witnessing.rs

```diff
@@ -290,7 +334,8 @@ async fn get_program_swap_event_accounts_data(
 				Err(anyhow!("Expected UiAccountData::Binary(String, UiAccountEncoding::Base64)")),
 			None => Ok((account, None)),
 		})
-		.collect()
+		.collect();
+	program_swap_event_accounts_iter
```

Comment:
why this change?

Reply:
oh, it's just because I had added some printing before. Will revert.